website/docs/r/alert_configuration.html.markdown

```diff
@@ -195,7 +195,7 @@ List of notifications to send when an alert condition is detected.
 * `api_token` - Slack API token. Required for the SLACK notifications type. If the token later becomes invalid, Atlas sends an email to the project owner and eventually removes the token.
 * `channel_name` - Slack channel name. Required for the SLACK notifications type.
 * `datadog_api_key` - Datadog API Key. Found in the Datadog dashboard. Required for the DATADOG notifications type.
-* `datadog_region` - Region that indicates which API URL to use. Accepted regions are: `US`, `EU`. The default Datadog region is US.
+* `datadog_region` - Region that indicates which API URL to use. Accepted regions are: `US`, `EU`, `US3`, `US5`. The default Datadog region is US.
```

Comment:
@jfmainville thank you very much for your contribution! We also need to update the `d` documentation:

* website/docs/d/alert_configurations.html.markdown
* website/docs/d/alert_configuration.html.markdown

Reply:
@marcosuma Tanks for the feedback! I added the missing references in the documentation for the `datadog_region` attribute